### ISSUE TITLE
Pino specific icons should be located in {$prefix}/share/pino/icons/

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#64]
```
Pino specific icons, like the ones used for direct reply, retweet and et
cetera should be located in
{$prefix}/share/pino/icons/hicolor/<size>/<context>, instead of using the
{$prefix}/share/icons/hicolor.

All the details of this standard can be found here:

http://live.gnome.org/ThemableAppSpecificIcons

Thank you.
```

Original issue reported on code.google.com by `vdepiz...@gmail.com` on 1 Feb 2010 at 9:41

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 1 Feb 2010 at 9:50
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by b3283d1b52 revision.
```

Original comment by `tro...@gmail.com` on 5 Feb 2010 at 1:10
- Changed state: **Fixed**
